js/runner.go

```diff
@@ -756,8 +757,50 @@ func (u *VU) runFn(
 	if u.Runner.Bundle.Options.NoVUConnectionReuse.Bool {
 		u.Transport.CloseIdleConnections()
 	}
+	// TODO move this to a function or something
+	builtinMetrics := metrics.GetBuiltInMetrics(ctx)
```

Comment:
This also was the easier way then to just keep adding to the Trail's "constuctor"

Reply:
As I mentioned in another comment, it should be relatively easy to thread the metric registry and the built-in metrics struct to the VU `State`, to which you already have direct access here.